### DISCUSSION TITLE
No reward after: 🍀 Farmed unfinished_block

### DISCUSSION THREAD

**Discussion by long-pham** [Chia-Network/chia-blockchain#7063] [Support]
I have ~7k plots but haven't won for 29 days.  Today I saw this in the log file:

`2021-06-27T09:05:05.145 full_node chia.full_node.full_node: INFO     🍀 ️Farmed unfinished_block bc07be0c9c13ded2ccc7a9849614ae4692bd7f945362730fe57805319558fb83, SP: 17, validation time: 0.057682037353515625, cost: 172875678`

Then: 

```
2021-06-27T18:46:05.329 wallet chia.wallet.wallet_state_manager: INFO     Adding coin: {'amount': 1650000000000,
 'parent_coin_info': '0xe302963c8937ab79618f8d7b043e9ba77fa5039db4f21d38e4bb71d81d8dae4a',
 'puzzle_hash': '0x80429798b1dcb6dedb082010c0f16095f14e3a3eb77ec6dc069ea10502dde315'} at 494006
2021-06-27T18:46:05.330 wallet chia.wallet.wallet_state_manager: INFO     Coins removed [Coin(parent_coin_info=<bytes32: ccd5bb71183532bff220ba46c268991a00000000000000000000000000047878>, puzzle_hash=<bytes32: 1b9acbc5002b28af26b0d1e0a6ccfe1cd05ceadc67aa0a82ffaf44393c6c1cd3>, amount=1750000000000)] at height: 494006
```

Why was the coin removed? is it the reason why I didn't win a block?

FYR, here a similar log that actually won a block:
`2021-05-24T14:25:29.711 full_node chia.full_node.full_node: INFO     🍀 ️Farmed unfinished_block 9872dfcadfe5b9a238d62087bc06734b2fb3fe4cbb464cbcc85828d281cbc9f3, SP: 39, validation time: 0.02488994598388672, cost: 65633792`

```
2021-05-24T14:26:27.048 wallet chia.wallet.wallet_state_manager: INFO     Adding coin: {'amount': 250000050000,
 'parent_coin_info': '0x3ff07eb358e8255a65c30a2dce0e5fbb00000000000000000000000000050ddd',
 'puzzle_hash': '0x1b9acbc5002b28af26b0d1e0a6ccfe1cd05ceadc67aa0a82ffaf44393c6c1cd3'} at 331232
2021-05-24T14:26:27.051 wallet chia.wallet.wallet_state_manager: INFO     Adding coin: {'amount': 1750000000000,
 'parent_coin_info': '0xccd5bb71183532bff220ba46c268991a00000000000000000000000000050ddd',
 'puzzle_hash': '0x1b9acbc5002b28af26b0d1e0a6ccfe1cd05ceadc67aa0a82ffaf44393c6c1cd3'} at 331232
```

Appreciate any help!

PS: double-checked: farmer xch_target_address is correct.

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
